plugins/woocommerce/templates/loop/pagination.php

```diff
@@ -12,17 +12,20 @@
  *
  * @see     https://woocommerce.com/document/template-structure/
  * @package WooCommerce\Templates
- * @version 9.3.0
+ * @version 9.4.0
```

Comment:
The expected version should be a corresponding WooCommerce version in which the change lands. So in this case it should stay at 9.3.0 as it was already bumped in https://github.com/woocommerce/woocommerce/pull/49924.

```suggestion
 * @version 9.3.0
```